src/main/resources/application-micrometer.yml

```diff
@@ -0,0 +1,39 @@
+project-contributors:
+  - marcingrzejszczak
+  - jonatan-ivanov
+  - shakuzen
+  - ttddyy
+
+issuebot:
+  monitoring:
+    repositories:
+      - organization: micrometer-metrics
+        name: micrometer
+        collaborators: ${project-contributors}
+      - organization: micrometer-metrics
+        name: tracing
+        collaborators: ${project-contributors}
+      - organization: micrometer-metrics
+        name: context-propagation
+        collaborators: ${project-contributors}
+      - organization: micrometer-metrics
+        name: micrometer-docs-generator
+        collaborators: ${project-contributors}
+  triage:
```

Comment:
Are the rest of these different?  Shouldn't you only need to specify the properties that are different in the profile specific configuration?